### DISCUSSION TITLE
Unable to get the benefit from GitHub Education Pack

### DISCUSSION THREAD

**Discussion by ramavats** [community/community#116718] [GitHub Education]
### Hi everyone,

Hello everyone, I am currently a student but unable to get any benefits of Github Education Pack.
I see this when I open https://education.github.com/benefits

![image](https://github.com/community/community/assets/22230571/dc07c41e-82f7-4714-b8bd-601bc51834bf)

Someone please help me.

**Comment by ralfaro17** [comment]
Student licenses usually last for 2 years, however, from time to time GitHub might ask again for proof that you are still a student from an institution, and if that's the case, you can reapply for the student pack by sending your student ID or any other document that shows you're still enrolled
